swan-lake/development-tutorials/source-code-dependencies/style-guide/top-level-definitions.md

```diff
@@ -42,18 +42,62 @@ intro: The sections below include the coding conventions with respect to top-lev
 
 * Do not keep spaces between the organization name, divider `/`, and module name.
 
-    **Example,**
+    **Example:**
 
     ```ballerina
     import ballerina/http;
     ```
 
 * Imports should be sorted alphabetically, first by the organization name and then by the module name.
 
+## Constant declaration
+
+* Use the SCREAMING_SNAKE_CASE for constant names.
+
+    **Example:**
+    
+    ```ballerina
+    // don't
+    const minAge = 20;
+    const MinAge = 20;
+    const min_age = 20;
+    
+    // do
+    const MIN_AGE = 20;
+    ```
+
+* It is recommended to define the constants without the type descriptor, unless the type descriptor is used as the inherent type.
```

Comment:
Shall we add an example for the case mentioned by "the type descriptor is used as the inherent type" under the `Dos` section? It would make the point clearer to the user.

Reply:
It seems default values are not supported either.
```bal
const Person P1 = {name: "Frank"};

type Person record {|
    int age = 21;
    string name;
|};
```

Can't think of an example for the moment.